docs/auth/tokens.rst

```diff
@@ -482,6 +501,55 @@ and you cannot remove a default policy when other policies are still in place.
 During deletion of tokens, users, or domains, policies are cleaned up
 automatically.
 
+
+User Override
+`````````````
+One user can authorize another such that the latter can use their token to
+perform actions in the name of the former.
+For example, Alice can authorize Bob to use his (Bob's) token to act within her
+(Alice's) account.
+
+To this end, the email address associated with Alice's account needs to be set
+in the ``user_override`` field of Bob's token, after which the token is called
+an "override token".
+(Note that at this time, this feature is under development, and write access to
+this field is not available.)
+
+This construction allows Bob to act in Alice's name without requiring Alice to
+share any secrets with Bob. (Bob can use his own secret token.)
+
+It is particularly useful when combined with the ``perm_create_domain`` and
+``perm_delete_domain`` permissions, as well as the ``auto_policy`` flag:
+In this case, Bob will be able to create, manage (due to ``auto_policy``) and
+delete domains in Alice's account, without being able to see or modify other
+domains (or even tokens) that Alice might have in her account.
+(When ``auto_policy`` is not set, visibility of domains is not restricted.)
+
+When listing tokens, the configuration of override tokens is visible to both
+their owner and the user listed in the ``user_override`` field.
+To discern which tokens are (or are not) override tokens, associated email
+addresses are listed in both the ``owner`` and ``user_override`` fields.
+Note that as a result, both parties can observe when either deSEC account email
+address changes.
+
+Only API tokens without the ``perm_manage_tokens`` permission are eligible to
+become override tokens.
+Further, only the override user (not the token owner) can manage an override
+token, including changing its name or permissions.
+However, both the owner and the override user can delete an override token.
```

Comment:
We may need a UI warning for this

Reply:
done

![image](https://github.com/user-attachments/assets/55468fba-eab7-49d8-ba64-cf75397b6904)